Sources/ApolloCodegenLib/Templates/FragmentTemplate.swift

```diff
@@ -2,7 +2,7 @@ struct FragmentTemplate {
 
   let fragment: IR.NamedFragment
   let schema: IR.Schema
-  let config: ApolloCodegenConfiguration
+  let config: ApolloCodegenConfiguration.FileOutput
```

Comment:
These changes are narrowing the surface area of the configuration struct that gets passed around.